public/framework/core_refs.ts

```diff
@@ -3,7 +3,13 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { ChromeStart, HttpStart, IToasts } from '../../../../src/core/public';
+import {
+  ApplicationStart,
+  ChromeStart,
+  HttpStart,
+  IToasts,
+  IUiSettingsClient,
```

Comment:
are we using `IUiSettingsClient` anywhere?

Reply:
Missed this - was testing out to see about reading feature flag values, but that is not priority right now, thanks for the catch!